jina/clients/base/helper.py

```diff
@@ -30,7 +26,7 @@ def __init__(
         initial_backoff: float = 0.5,
         max_backoff: float = 0.1,
         backoff_multiplier: float = 1.5,
-        tracer_provider: Optional[trace.TracerProvider] = None,
+        tracer_provider: Optional[Any] = None,
```

Comment:
why Any? it should not be any

Reply:
I'm using Any for now to check if hiding imports will help in any way. I understand module level imports should also be hidden but also in some cases this import here is purely for typing info and nothing else.

Reply:
u can do the typing by hiding in 'trace.TraceProvider', just the string